ivadomed/models.py

```diff
@@ -769,3 +769,37 @@ def __init__(self, in_size, out_size, kernel_size=(1, 1, 1), is_batchnorm=True):
     def forward(self, inputs):
         outputs = self.conv1(inputs)
         return outputs
+
+def set_model_for_retrain(m, retrain_fraction):
+    """Set model for transfer learning.
+
+    The first layers (defined by 1-retrain_fraction) are frozen (i.e. requires_grad=False).
+    The weights of the last layers (defined by retrain_fraction) are reset.
+    Args:
+        m (torch module): pretrained model.
```

Comment:
For code clarity, maybe the variable m, n, p and l could be changes to more descriptive var names e.i layer or model. Your call :)

Reply:
eheh yep, happy to change this :)